pkg/config/namespaces.go

```diff
@@ -0,0 +1,186 @@
+// Code generated by pkg/config/generate/namespaces.go; DO NOT EDIT.
+// Generated at 2021-11-19 21:40:29.563134165 +0000 UTC
+package config
+
+var ConfigMapSources = []string{
+	"openshift-monitoring/addons-namespaces",
+	"openshift-monitoring/managed-namespaces",
+	"openshift-monitoring/ocp-namespaces",
+}
+
+var PrivilegedNamespaces = []string{
+	"^redhat.*",
```

Comment:
Do we need the regex anchors at all anymore since this are just fixed strings now?

Reply:
The regex is a holdover from [the current implementation](https://github.com/openshift/managed-cluster-validating-webhooks/blob/master/pkg/webhooks/namespace/namespace.go#L22). 

I don't have any qualms about removing the regex per se, but my understanding was that we'd like to keep the current behavior with regard to namespaces that are prefixed with `redhat`. I believe this is because the current namespace lists aren't comprehensive enough to cover all layered product namespaces, which still need protection from webhooks.

I can't think of any way to accomplish both, but if anyone has suggestions, I'm certainly open to improvements

Reply:
Ah that makes sense, I didn't see the redhat wildcard regex.